src/connector/src/connector_common/iceberg/mod.rs

```diff
@@ -85,6 +85,9 @@ pub struct IcebergCommon {
         deserialize_with = "deserialize_optional_bool_from_string"
     )]
     pub path_style_access: Option<bool>,
+    /// enable config load currently is used by iceberg engine, so it only support jdbc catalog.
+    #[serde(default, deserialize_with = "deserialize_optional_bool_from_string")]
+    pub enable_config_load: Option<bool>,
```

Comment:
Why not just a boolean flag instead of `option<bool>`

Reply:
option means this is not a required field, especially for user facing iceberg sink and source